modules/light-clients/11-beefy/types/update_test.go

```diff
@@ -354,16 +366,18 @@ func TestCheckHeaderAndUpdateState(t *testing.T) {
 				require.Equal(t, clientState.MmrRootHash, signedCommitment.Commitment.Payload, "failed to update client state. LatestBeefyHeight: %d, Commitment.BlockNumber %d", clientState.LatestBeefyHeight, uint32(signedCommitment.Commitment.BlockNumber))
 			}
 			t.Log("====== successfully processed justification! ======")
+			
+			count++
 
-			paramSpace := types2.NewSubspace(nil, nil, nil, nil, "test")
-			//paramSpace = paramSpace.WithKeyTable(clientypes.ParamKeyTable())
+			// paramSpace := types2.NewSubspace(nil, nil, nil, nil, "test")
+			// //paramSpace = paramSpace.WithKeyTable(clientypes.ParamKeyTable())
 
-			k := keeper.NewKeeper(nil, nil, paramSpace, nil, nil)
-			ctx := sdk.Context{}
-			store := k.ClientStore(ctx, "1234")
+			// k := keeper.NewKeeper(nil, nil, paramSpace, nil, nil)
+			// ctx := sdk.Context{}
+			// store := k.ClientStore(ctx, "1234")
 
-			err = clientState.UpdateState(sdk.Context{}, nil, store, &header)
```

Comment:
is this supposed to be removed ?

Reply:
it was failing for me with nill derefrencing

Reply:
cool, so I made this optional then.